src/sqlfluff/core/default_config.cfg

```diff
@@ -101,6 +101,12 @@ spacing_after = touch
 [sqlfluff:layout:type:end_square_bracket]
 spacing_before = touch
 
+[sqlfluff:layout:type:start_angle_bracket]
+spacing_after = touch
+
+[sqlfluff:layout:type:end_angle_bracket]
+spacing_before = touch
```

Comment:
As discussed it is possible to make this an optional `touch`? The following should not touch before `LANGUAGE` for example:

```sql
CREATE TEMPORARY FUNCTION DoSomething(param1 STRING, param2 STRING)
RETURNS ARRAY<STRING> LANGUAGE js AS """Some JS""";

SELECT DoSomething(col1) FROM table1
```

Big query does seem to accept it without the space but it looks weird.

Reply:
That config applies to the spacing between `STRING` and `>`. The problematic one for that query is the `spacing_after` for `[sqlfluff:layout:type:array_type]`, as discussed via slack I've updated that to be more liberal for now until we have a better option.